### PR TITLE
JPA and Flyway

### PR DESCRIPTION
**Pull request by katonaaron-msg**
- Add initial application
- Create data model: JPA and Flyway

### CODE REVIEW

File: mvnw [hunk 1]
```diff
@@ -0,0 +1,308 @@
+#!/bin/sh
```
Comment (cristinaAnca) [inline mvnw]: Please exclude these maven files from git (add them in git ignore)

File: src/main/java/ro/msg/learning/shop/ShopApplication.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package ro.msg.learning.shop;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@SpringBootApplication
+@EnableTransactionManagement
```
Comment (cristinaAnca) [inline src/main/java/ro/msg/learning/shop/ShopApplication.java]: Why do you need this annotation?

**Review by cristinaAnca — CHANGES_REQUESTED** [review]

File: mvnw (hunk 1 above)
Comment (katonaaron-msg) [reply mvnw]: Removed and excluded the wrapper files.

File: src/main/java/ro/msg/learning/shop/ShopApplication.java (hunk 2 above)
Comment (katonaaron-msg) [reply src/main/java/ro/msg/learning/shop/ShopApplication.java]: Removed the annotation

**Review by cristinaAnca — APPROVED** [review]

**Pull request merged**